src/promptflow-evals/tests/unittests/test_evaluate_unittest.py

```diff
@@ -0,0 +1,81 @@
+# ---------------------------------------------------------

```

Comment:
Sorry if my previous comment is not clear. 
Please move the tests here to src/promptflow-evals/tests/evals/unittests/test_evaluate.py
Correspondingly, move test data to src/promptflow-evals/tests/evals/unittests/data

Nakunmar's PR will remove "src/promptflow-evals/tests/unittests"

Reply:
Fixed.